docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -88,9 +76,9 @@ Inside the ``vertiq_io`` submenu one option should appear for including IFCI con
 
     Include IFCI Parameters
 
-Once this is selected, a second option will appear for including pulsing module configurations. If you plan on using `underactuated propellers <https://www.vertiq.co/up12>`_, select this as well.
+When IFCI Configuration Paraeters is selected, a second option will appear for including pulsing module configurations. If you plan on using `underactuated propellers <https://www.vertiq.co/up12>`_, select this as well.
```

Comment:
paraeters typo

Reply:
fixed in [0e75000](https://github.com/iq-motion-control/iq-module-communication-doc/pull/103/commits/0e75000edabd362ec6f3b56d5f5fa48c3bb53580)